spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_helper'
+require 'helpers/models_helper'
+
+feature 'User Index' do
+  background do
+    @name1 = 'user1'
+    @user1 = create_and_activate_user(@name1)
+    @name2 = 'user2'
+    @user2 = create_and_activate_user(@name2)
+
+    visit users_path
+  end
+
+  scenario 'shows username of all the users' do
+    expect(page).to have_content @name1
+    expect(page).to have_content @name2
+  end
+
+  scenario 'shows photos of all the users' do
+    expect(page).to have_css "img[src='#{@name1}.jpg']"
+    expect(page).to have_css "img[src='#{@name2}.jpg']"
+  end
+
+  scenario 'shows zero posts if users have no posts' do
+    expect(page).to have_content 'Number of posts: 0'
+  end
+
+  scenario 'shows number of posts if users have posts' do
+    @user1.posts_counter = 1
+    @user2.posts_counter = 5
+    [@user1, @user2].each(&:save)
+
+    visit users_path
+
+    expect(page).to have_content 'Number of posts: 1'
+    expect(page).to have_content 'Number of posts: 5'
+  end
+
+  scenario 'gets redirected to user page after click on it' do
+    click_link @name1
+
+    expect(page).to have_current_path(user_path(@user1))
+  end
+end
```

Comment:
- [ ] Kindly fix the error.
![image](https://user-images.githubusercontent.com/38635158/174333278-a3c866ec-2c6a-41ea-b719-15666a1c1833.png)